batavia/core/builtins.js

```diff
@@ -348,8 +348,42 @@ batavia.builtins.delattr = function(args, kwargs) {
 batavia.builtins.delattr.__doc__ = "delattr(object, name)\n\nDelete a named attribute on an object; delattr(x, 'y') is equivalent to\n``del x.y''.";
 
 batavia.builtins.dict = function(args, kwargs) {
-    throw new batavia.builtins.NotImplementedError(
-        "Builtin Batavia function 'dict' not implemented");
+    if (args.length > 1) {
```

Comment:
isType is used in builtin sorted, taking that out will break the function.

Also instanceof is good enough only for checking custom types with custom constructors.
If you need to check JavaScript type this is totally broken. 
More about this with examples in this article:
http://bonsaiden.github.io/JavaScript-Garden/#types.equality

Should i Still remove isType util and rework sorted function in builtins?